src/features/ModelPlan/ChangeHistory/util.tsx

```diff
@@ -54,6 +54,7 @@ export type TranslationTables =
   | TableName.PLAN_OPS_EVAL_AND_LEARNING
   | TableName.PLAN_PAYMENTS
   | TableName.PLAN_COLLABORATOR
+  | TableName.PLAN_DATA_EXCHANGE_APPROACH
```

Comment:
Is this including some of the changes from https://github.com/CMS-Enterprise/mint-app/pull/1440 ?

Fine if so, just wanted to make sure I understood the scope of the diff here!

Reply:
Yep, I had a comment at the top of the PR noting this.